global_data.py

```diff
@@ -32,33 +32,53 @@ class WpReq(Enum):
 
 
 solver_state_items = [
-    ("OKAY", "Okay", "Successfully solved sketch", "CHECKMARK", 0),
+    (
+        "OKAY",
+        "Okay",
+        "Successfully solved sketch",
+        "CHECKMARK",
+        0,  # SLVS_RESULT_OKAY
+    ),
     (
         "INCONSISTENT",
         "Inconsistent",
         "Cannot solve sketch because of inconsistent constraints",
         "ERROR",
-        1,
+        1,  # SLVS_RESULT_INCONSISTENT
     ),
     (
         "DIDNT_CONVERGE",
         "Didnt Converge",
         "Cannot solve sketch, system didn't converge",
         "ERROR",
-        2,
+        2,  # SLVS_RESULT_DIDNT_CONVERGE
     ),
     (
         "TOO_MANY_UNKNOWNS",
         "Too Many Unknowns",
         "Cannot solve sketch because of too many unknowns",
         "ERROR",
-        3,
+        3,  # SLVS_RESULT_TOO_MANY_UNKNOWNS
+    ),
+    (
+        "INIT_ERROR",
+        "Initialize Error",
+        "Initializing solver failed",
+        "ERROR",
+        4,  # SLVS_RESULT_INIT_ERROR
+    ),
+    (
+        "REDUNDANT_OK",
+        "Redundant Ok",
+        "Constraints seem redundand",
+        "CHECKMARK",
+        5,  # SLVS_RESULT_REDUNDANT_OK
     ),
     (
         "UNKNOWN_FAILURE",
```

Comment:
Do we still need the unknown failiure? I've used this because the solver was sometimes returning an undocumented value. Since you've looked at the source, is there a case where the solver returns the value 6?

Reply:
Need them - strickly no. Then again if in the future the library decides to provide other return states you can pretty fast notice because users will report "Unknown failure".

Reply:
Makes sense...